doc/development_guide/contribute.rst

```diff
@@ -166,6 +166,6 @@ A Typical Development Workflow
    It is also possible to give tox a `pytest specifier <https://docs.pytest.org/en/latest/usage.html#specifying-tests-selecting-tests>`_
    to run only your test::
 
-     $ tox pylint/test/test_functional.py::test_functional
+     $ tox -e py36 -- -k 'test_functional[implicit_str_concat_in_sequence]'
```

Comment:
Why did you modify this?

Reply:
To help future contributors by showing more useful CLI flags:
- the ability to execute the tests in only one tox environment
- the ability to execute only one functional test